src/chrome/chromeDebugAdapter.ts

```diff
@@ -1305,6 +1349,21 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
         };
     }
 
+    private scriptToScriptEvent(script: Crdp.Debugger.ScriptParsedEvent): ScriptEvent {
+        const sourceReference = this.getSourceReferenceForScriptId(script.scriptId);
+        const origin = this.getReadonlyOrigin(script.url);
+        const source = {
+            name: path.basename(script.url),
+            path: script.url,
+            sourceReference,
+            origin
+        };
+
+        const pinezorroScript: Script = new Script(source);
```

Comment:
Don't refer to pinezorro here. It's relevant for the debug protocol itself, not a particular client